OpenDreamRuntime/DreamManager.cs

```diff
@@ -107,6 +108,10 @@ public sealed partial class DreamManager {
             if (json == null)
                 return false;
 
+            if (!json.Metadata.Version.Equals(OpcodeVerifier.GetOpcodesHash())) {
+                _sawmill.Warning("Compiler opcode version does not match the runtime version! This may cause issues!");
```

Comment:
I chose a warning because it /could/ be intentional? This could be upgraded to an error if wanted.

Reply:
In what cases could running a json from an old bytecode version be intentional?

Reply:
Spent the day thinking about it, couldn't really think of anything. If someone is really determined to try this though, they can just comment this out.

Reply:
An error log by itself doesn't stop the program anyways